cmake/Modules/FindAravis.cmake

```diff
@@ -1,5 +1,5 @@
 include(ToolchainLibraryFinder)
-ToolchainLibraryFinder(
+toolchainlibraryfinder(
```

Comment:
`cmake-format` shouldn't have done this. Have you merged in master recently?

Reply:
I have just merged master in, although I only branched off master a few days ago

Reply:
https://github.com/NUbots/NUbots/blob/hamiltons/fix_build_warnings/.cmake-format.py#L233
That line in `cmake-format` should prevent it from messing with the case of ToolchainLibraryFinder. The fact that it either means you're using the wrong version of the docker image (run `./b target generic`) or something else has gone wrong

Reply:
that worked. Had to rebuild a bunch, but ./b format fixed the capitalisation back I think

Reply:
Need to commit the changes to undo. It will just not change the case, not fix it back to the correct case